subgraphs/uniswap-forks/protocols/ubeswap/config/networks/celo/celo.ts

```diff
@@ -0,0 +1,92 @@
+import { Address, BigDecimal } from '@graphprotocol/graph-ts';
+import { Factory } from '../../../../../generated/Factory/Factory';
+import { FeeSwitch, Network, PROTOCOL_SCHEMA_VERSION, RewardIntervalType } from '../../../../../src/common/constants';
+import { Configurations } from '../../../../../configurations/configurations/interface';
+import { PROTOCOL_SUBGRAPH_VERSION, PROTOCOL_METHODOLOGY_VERSION, PROTOCOL_NAME, PROTOCOL_SLUG } from '../../../src/common/constants';
+import { toLowerCase, toLowerCaseList } from '../../../../../src/common/utils/utils';
+
+export class UbeswapCeloConfigurations implements Configurations {
+  getNetwork(): string {
+    return Network.CELO;
+  }
+  getProtocolName(): string {
+    return "Ubeswap";
+  }
+  getProtocolSlug(): string {
+    return "Ubeswap";
```

Comment:
can you change to all lowercase: `ubeswap`?

Reply:
ok